platforms/web/.env

```
@@ -1,12 +1,12 @@
 APP_API_BASE_URL=https://cdn.jwplayer.com
 APP_PLAYER_ID=M4qoGvUk
 
-
 ### Web-only env vars (not sent to common/src/env configureEnv())
-APP_FOOTER_TEXT=\u00a9 JW Player | [jwplayer.com](https://www.jwplayer.com/)
+APP_FOOTER_VERSION=$npm_package_version
```

```suggestion
APP_VERSION=$npm_package_version
```

Small remark: I'd suggest omitting the 'FOOTER' text here, to only describe what it is, namely the app version

---

We've internally discussed that the `APP_VERSION` is already used throughout the project, therefore we cannot use it in this instance. So I'd say change it to `FOOTER_APP_VERSION` to make it clear it is about the version of the app.

---

I've indeed mentioned that we are already using the `APP_VERSION`, but let's try merging these instead of playing safe 😄 

I have tested this; 

- Adding `APP_VERSION` to the `.env` file
- Removing the redundant  `'import.meta.env.APP_VERSION': JSON.stringify(process.env.npm_package_version),` in platforms/web/vite.config.ts